compose.yaml

```diff
@@ -0,0 +1,12 @@
+version: '1'
+
+services:
+  fastapi-app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    image: infinity-api:latest
+    ports:
+      - "3000:3000"
+    env_file:
+      - .env
```

Comment:
The code only worked for me once I created a `.env` file in the repo. 
I think this could be described on the README, although the solution was easy.

Reply:
I think this will do https://github.com/RocketPy-Team/Infinity-API/pull/13/commits/639b749d11d5a65630fd3b98988fc0e98d455571